tests/test_ringsSkyMap.py

```diff
@@ -106,6 +107,11 @@ def testDm14809(self):
         tract = self.skymap.findTract(coord)
         self.assertTrue(tract.contains(coord))
 
+        tractId = self.skymap.findTractIdArray(np.array([coord.getLongitude().asDegrees()]),
```

Comment:
Consider leaving ra/dec as scalars to ensure that the `np.atleast_1d` call works as intended.